tests/unit-test/profile.ts

```diff
@@ -2,86 +2,124 @@ import { CONFIG, wait } from '../common_config'
 import { installAgents } from '../installAgents'
 
 module.exports = (orchestrator) => {
+ orchestrator.registerScenario('test profile zomes', async (s, t) => {
+  // spawn the conductor process
+  const [conductor] = await s.players([CONFIG])
+  let [alice_happ, bobbo_happ] = await installAgents(conductor, [
+   'alice',
+   'bobbo',
+  ])
```

Comment:
how does passing `is_editable_profile` as undefined here work? I'm missing where it defaults to true. Tests are passing, so obviously it works, but I'd like to understand how.